### DISCUSSION TITLE
Support for Code Generation on Standalone SPM Package

### DISCUSSION THREAD

**Discussion by baksha97** [birdrides/mockingbird#236] [General]
I'm a little confused. Is it possible to utilize this framework for creating mocks on standalone packages? Packages only use a ```Package.swift``` file, there are no build scripts or targets that you can configure the build scripts on. Does anyone have any input on these this?

Thanks!

**Comment by andrewchang-bird** [comment]
The latest release (0.19) now ships with improved support and guidance for SwiftPM packages. Here’s the [quick start guide](https://mockingbirdswift.com/spm-package-quickstart) with more info.
